src/utils/kubernetes-manager.ts

```diff
@@ -12,27 +13,48 @@ export class KubernetesManager {
 
   constructor() {
     this.kc = new k8s.KubeConfig();
-    this.kc.loadFromDefault();
+    if (this.isRunningInCluster()) {
+      console.log("Running inside a Kubernetes cluster");
+      this.kc.loadFromCluster();
+    } else {
+      console.log("Loading default kube config file");
+      this.kc.loadFromDefault();
+    }
```

Comment:
```suggestion
    if (this.isRunningInCluster()) {
      this.kc.loadFromCluster();
    } else {
      this.kc.loadFromDefault();
    }
```

I think some clients had issues if you put stdout that isn't valid jsonrpc

Reply:
Updated to remove console.log statements.